src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -4932,6 +4961,11 @@ class CgroupUtils(object):
             pbs.logmsg(pbs.EVENT_DEBUG4, '%s: Job %s is not running' %
                        (caller_name(), jobid))
             return
+        if pbs.event().type == pbs.EXECHOST_PERIODIC or \
+           pbs.event().type == pbs.EXECJOB_EPILOGUE:
```

Comment:
Consider: if pbs.event().type in [pbs.EXECHOST_PERIODIC, pbs.EXECJOB_EPILOGUE]: 

here and below